cli/src/main/resources/nls/Help.properties

```diff
@@ -63,9 +63,9 @@ cmd.kotlincnative.detail=Kotlin/Native Compiler (kotlincnative) compiles Kotlin
 cmd.lazydocker=Tool commandlet for LazyDocker.
 cmd.lazydocker.detail=Lazydocker is a simple terminal UI for both docker and docker-compose. Detailed documentation can be found at https://github.com/jesseduffield/lazydocker
 cmd.list-editions=List the available editions of the selected tool.
-cmd.list-editions.detail=TODO list-editions
+cmd.list-editions.detail=To list all available editions of f.e. 'intellij' simply type: 'ide list-editions intellij'. IDEasy will then list 'ultimate' and 'intellij' (community edition).
```

Comment:
Never redundantly put such information into localized help messages. If we would add a new edition, to `intellij`, we would very likely forget to update this.
I very much like your idea to add web-links to further details like you have done for Java.
It would be lovely if we would do that for all our tools because this could really be helpful for the end-user.
In Git-Bash / WT the user can control-click the link to open in the browser...

Reply:
I've removed the specific information about the intellij versions now.